source/MaterialXTest/GenOsl.h

```diff
@@ -39,6 +39,11 @@ class OslShaderGeneratorTester : public GenShaderUtil::ShaderGeneratorTester
         ParentClass::addSkipNodeDefs();
     }
 
+    // Arnold specific files are ignored in vanilla osl target
+    void addSkipLibraryFiles() override
+    {
+        _skipLibraryFiles.insert( "pbrlib_genosl_arnold_impl.mtlx" );
```

Comment:
I'm a bit confused as to how this situation is run into as this file is only installed if you turn on Arnold code generation testing AFAIK.

Reply:
Yes but since it is [on by default](https://github.com/autodesk-forks/MaterialX/blob/adsk_contrib/dev/CMakeLists.txt#L54) it is failing ... by default without this change.